devops/deploy/spin-up-script.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+# Just a basic script to spin up listening servers on different ports
+PORTS=$(seq 30000 30050)
+
+export PYTHONPATH=../../src/main/py
+
+for port in ${PORTS}
+do
+  echo "Starting on port $port"
+  python3 -m sample_module.udp_server ${port} &
+done
+
```

Comment:
We'll probably want a spin-down script or function a well :-)

Reply:
Should be okay for now since we don't have any custom shutdown logic, especially if we use the more "gentle" signals. If we end up with something that needs graceful shutdown, we'll need to implement the appropriate signal handlers. For example:
```
import signal
...
class Foo:
  def __init__(self):
    signal.signal(signal.SIGTERM, self.clean_shutdown)
...
```